### PR TITLE
[master<] Callable mappings path

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Added docs for a new config flag that allows you to map procedures

### CODE REVIEW

**Comment by vpavicic** [comment]
@gvolfing i'm not sure about the title in the load-call-query-modules.md.. what do you think?

**Comment by vpavicic** [comment]
> > @gvolfing i'm not sure about the title in the load-call-query-modules.md.. what do you think?
> 
> Rereading this, I think we miss some context for the user to completely understand what this flag is used for. As for the title I think it should be something like `Mapping costume procedure names to existing query procedures.` and the body of the section should also clarify that only the names will be mapped, nothing fancy. So something like `If you want to use available query procedures with a different name, you can specify the mapping from your desired new procedure names to the already existing query procedures. You can define the mapping of Memgraph query modules or advanced algorithms in a JSON file, then define the path as the value of the `query-callable-mappings-path`[configuration flag](/reference-guide/configuration.md). ` I should have clarify this yesterday, my bad! @vpavicic Let me know what you think!

I'm so sorry Gabor, I C/P the wrong explanation :(

We agreed on this, are we still ok with this? I'll use your title

 If you want to replace procedure names your application calls without changing
  the application code, you can define the mapping of the old and new procedure
  names in a JSON file, then set the path to the files as the value of the
  `query-callable-mappings-path` [configuration
  flag](/reference-guide/configuration.md).

**Comment by gvolfing** [comment]
> > > @gvolfing i'm not sure about the title in the load-call-query-modules.md.. what do you think?
> > 
> > 
> > Rereading this, I think we miss some context for the user to completely understand what this flag is used for. As for the title I think it should be something like `Mapping costume procedure names to existing query procedures.` and the body of the section should also clarify that only the names will be mapped, nothing fancy. So something like `If you want to use available query procedures with a different name, you can specify the mapping from your desired new procedure names to the already existing query procedures. You can define the mapping of Memgraph query modules or advanced algorithms in a JSON file, then define the path as the value of the `query-callable-mappings-path`[configuration flag](/reference-guide/configuration.md). ` I should have clarify this yesterday, my bad! @vpavicic Let me know what you think!
> 
> I'm so sorry Gabor, I C/P the wrong explanation :(
> 
> We agreed on this, are we still ok with this? I'll use your title
> 
> "If you want to replace procedure names your application calls without changing the application code, you can define the mapping of the old and new procedure names in a JSON file, then set the path to the files as the value of the `query-callable-mappings-path` [configuration flag](/reference-guide/configuration.md)."

@vpavicic That's perfect!

File: docs/reference-guide/query-modules/load-call-query-modules.md [hunk 1]
```diff
@@ -23,6 +23,21 @@ read [the query modules overview page](overview.md).
...
+names in a JSON file, then set the path to the files as the value of the
+`query-callable-mappings-path` [configuration
+flag](/reference-guide/configuration.md). 
+
+Example of a JSON file:
+
+{
+    "db.components": "mgps.components",
+    "util.validate": "mgps.validate"
+}
```
Comment (vpavicic) [inline docs/reference-guide/query-modules/load-call-query-modules.md:39]:
```suggestion
```json
{
    "db.components": "mgps.components",
    "util.validate": "mgps.validate"
}
```
```

**Pull request merged**